### PR TITLE
[Deprecated] Add @Deprecated annotations

### PR DESCRIPTION
**Pull request by makrushinoff**

### CODE REVIEW

File: src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java [hunk 1]
```diff
@@ -1,7 +1,7 @@
 package ua.kpi.iasa.IASA_Organiser.data.impl;
 
 import ua.kpi.iasa.IASA_Organiser.model.Event;
-
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java:4]: Add java docs here

File: src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+@Deprecated
 public class EventStore {  //smth like database/it`s foundation
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java:5]: Remove redundant comment;
(Never use write comments with '//', write TODO or better java docs)

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java (hunk 2 above)
Comment (MLGMAG) [reply src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java:8]: Fix it pls

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java [hunk 3]
```diff
@@ -2,7 +2,11 @@
 
 import ua.kpi.iasa.IASA_Organiser.model.Event;
 
-public class EventStore {  //smth like database/it`s foundation
+/**
+ * @deprecated It was useful at first lab, but now useless
```
Comment (MLGMAG) [inline src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java:6]: Change java docs comment to
"Class was used by {@link DefaultArrayDataManager}, but not that DataManager is deprecated and you don't need to use {@link EventStore} class"

**Review by MLGMAG — CHANGES_REQUESTED** [review]

File: src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java (hunk 3 above)
Comment (makrushinoff) [reply src/main/java/ua/kpi/iasa/IASA_Organiser/data/impl/EventStore.java]: Fixed

**Review by MLGMAG — APPROVED** [review]

**Review by VasiliyPetrov1 — APPROVED** [review]

**Pull request merged**